A3A/addons/core/functions/Base/fn_citySupportChange.sqf

```
@@ -5,7 +5,11 @@ if (!isServer) exitWith {Error("Server-only function miscalled")};
 waitUntil {!cityIsSupportChanging};
 cityIsSupportChanging = true;
 
-params ["_changeGov", "_changeReb", "_pos", ["_scaled", true], ["_isRadio", false]];
+params [["_changeGov",0,[0]], ["_changeReb",0,[0]], "_pos", ["_scaled", true], ["_isRadio", false]];
+if !(_changeGov isEqualType 0) exitWith {Error("The first parameter, the government support, must be a number"); "Error: The first parameter must be a number"};
+if !(_changeReb isEqualType 0) exitWith {Error("The second parameter, the rebel support, must be a number"); "Error: The second parameter must be a number"};
```

I don't think these can ever trigger because the params line forces the value?

---

From testing in both scheduled and unscheduled, it just silently replaces with the default value if the type isn't in the types array.